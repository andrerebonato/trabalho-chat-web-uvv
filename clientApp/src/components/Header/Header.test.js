import React from 'react';
import renderer from 'react-test-renderer';
import { HeaderComponent } from '../index';

describe('renders correctly HeaderComponent - isLoggedIn prop', () => {

    it('test snapshot when prop isLoggedIn = true', () => {
        const component = renderer.create(
            <HeaderComponent isLoggedIn />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});