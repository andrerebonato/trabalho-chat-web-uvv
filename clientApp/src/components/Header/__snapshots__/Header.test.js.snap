// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly HeaderComponent - isLoggedIn prop test snapshot when prop isLoggedIn = true 1`] = `
<header
  className="Header primary-bg"
>
  <nav
    className="Nav"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/"
    >
      Conversar
    </a>
    <a
      href="/"
    >
      Sobre
    </a>
    <button
      className="btn btn-danger"
    >
      Minha conta
    </button>
  </nav>
  <button
    className="Burger"
    onClick={[Function]}
  >
    Menu
  </button>
</header>
`;
